package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "0.6.6",
+  "version": "0.6.7",
```

Comment:
This will be 0.6.8 now that Andre's PR is merged.

Reply:
oh nice, thanks for the heads up. Will update!

Reply:
Updated